app/views/organizations/page_texts/_form.html.erb

```diff
@@ -3,34 +3,50 @@
     <div class="d-lg-flex align-items-center justify-content-between">
       <h3 class="mb-1">Landing page details</h3>
       <p class="text-muted mb-3" style="font-size: 12px;">Edit your landing page.
-      <p>
+      </p>
     </div>
     <div class="row mt-3">
       <div class="col-lg-6">
         <%= form.fields_for :page_text do |page_text_form| %>
-          <form class="mb-3">
-            <!-- Hero Text -->
-            <div class="form-group">
-              <%= form.text_field :hero,
-                                  label: "Hero",
-                                  autofocus: true,
-                                  placeholder: "hero text",
-                                  class: 'form-control' %>
-            </div>
-            <!-- About Us -->
-            <div class="form-group">
-              <%= form.text_area :about, label: "About Us", placeholder: "About Us text", class: 'form-control' %>
-            </div>
-          </form>
+          <div class="form-group">
+            <%= form.text_field :hero,
+                                label: "Hero",
+                                autofocus: true,
+                                placeholder: "hero text",
+                                class: 'form-control' %>
+          </div>
+          <div class="form-group">
+            <%= form.file_field :hero_image,
+                                label: "Hero Image",
+                                class: 'form-control'%>
+          </div>
+          <div class="form-group mt-4">
+            <%= form.text_area :about, label: "About Us", placeholder: "About Us text", class: 'form-control' %>
+            <small class="form-text text-muted">You can upload up to 2 images. </small>
```

Comment:
Can we move this text below the input instead? And can we also add the image spec requirements i.e., `Images must be .png or .jpeg under 2MB`